### ISSUE TITLE
More accessible themes

### ISSUE THREAD

**Issue by cyip92** [IvarK/AntimatterDimensionsSourceCode#3316]
**Is your feature request related to a problem? Please describe.**
It's been suggested a few times that the game should be more accessible to colorblind people and people who need screen readers. Given the vast span of different themes we have, many of which are pretty much impossible to make accessible in this way, it probably makes more sense to have a small handful of dedicated themes for colorblindness instead of attempting to make all of them able to support it.

For screen readers, it might make more sense to have an independent ON/OFF toggle in the options instead?

**Additional context**
We already have an older issue for this (#2016), but I felt the need to make a new one; I suspect that as long as post-release bugs remain unfixed, it'll be a really long time (if ever) until older issues are revisited, and I think this should be a higher priority than that.
